.gitignore

```diff
@@ -19,6 +19,8 @@ py27_gevent/
 py27_native/
 py3[0-9]_gevent/
 py3[0-9]_native/
+a.out
```

Comment:
Where does this even come from?

Reply:
My guess is the "compiler_ok_with_extra_std" function in our build process.

Reply:
Probably a good idea to also fix that. Adding a TODO for now would also be fine.